lib/transport.js

```diff
@@ -6,6 +6,27 @@ const { join, isAbsolute } = require('path')
 
 const ThreadStream = require('thread-stream')
 
+function setupOnExit (stream) {
+  /* istanbul ignore next */
+  if (global.WeakRef && global.WeakMap && global.FinalizationRegistry) {
```

Comment:
Looks like v14.17.3 is the first version with full support for all of these. Do we need to make this clear in the documentation?

Reply:
can we just release 7 for min Node version 14.7.3?

Reply:
I say remove the checks and do a version check in index.js, throw and say use Pino 6 for older node versions